scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -2,9 +2,14 @@ package org.scalameter
 
 
 
-import collection._
-import compat._
-import utils.withGCNotification
+import java.lang.reflect.Method
+import scala.collection._
+import scala.compat._
+import scala.runtime.BoxesRunTime
+import scala.util.matching.Regex
+import org.scalameter.execution.invocation.InvocationCountMatcher
```

Comment:
I was always thinking that I should place internal imports as the last :-)
So probably, they are unfortunately also in such order in other places when I was touching the code (but I'm not sure about it and don't know how to grep it easily)